mmdet/datasets/openimages.py

```diff
@@ -66,18 +71,41 @@ def __init__(self,
                  meta_file='',
                  filter_labels=True,
                  load_image_level_labels=True,
+                 file_client_args=dict(backend='disk'),
                  **kwargs):
+        # may get error if use other file_client
+        self.file_client_args = file_client_args
+
         self.cat2label = defaultdict(str)
         self.index_dict = {}
+
+        # Although it will init file_client in `CustomDataset`,
+        # it needs to be init here.
+        file_client = mmcv.FileClient(**file_client_args)
         # need get `index_dict` before load annotations
-        class_names = self.get_classes_from_csv(label_file)
-        super(OpenImagesDataset, self).__init__(**kwargs)
+        assert label_file.endswith('csv')
+        if hasattr(file_client, 'get_local_path'):
+            with file_client.get_local_path(label_file) as local_path:
+                class_names = self.get_classes_from_csv(local_path)
+        else:
+            class_names = self.get_classes_from_csv(label_file)
+        super(OpenImagesDataset, self).__init__(
+            ann_file=ann_file, file_client_args=file_client_args, **kwargs)
         self.CLASSES = class_names
         self.image_level_ann_file = image_level_ann_file
         self.load_image_level_labels = load_image_level_labels
         if get_supercategory is True:
-            assert hierarchy_file is not None
-            self.class_label_tree = self.get_relation_matrix(hierarchy_file)
+            assert hierarchy_file is not None and \
```

Comment:
It seems that the test failed because when the default value of `get_supercategory` is true, `hierarchy_file` must be passed in.